themes/feminicide-theme/layouts/_default/baseof.html

```diff
@@ -8,6 +8,13 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+    <meta property="og:title" content="Feminizidmap.org - A research project on femi(ni)cides in Germany" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="themes/feminicide-theme/static/bg-gradient-01-1280.png" />
```

Comment:
This needs to be an absolute link because it will be requested by external sites. You can use the `{{.Site.BaseURL}}` variable here, too

Reply:
oh nooo, again a brnach mess!!!!

Reply:
should i merge myri/publications then?

Reply:
I merged it